ibis/expr/types/relations.py

```diff
@@ -2756,7 +2808,7 @@ def fillna(
                         f"value of type {val_type} - pass in an explicit mapping "
                         f"of fill values to `fillna` instead."
                     )
-        return ops.FillNa(self, replacements).to_expr()
+        return self.fill_null(replacements)
```

Comment:
You can delete the full implementation above and just keep this line - no need to duplicate any of the `fill_null` function body in the old deprecated method.